system_tests/run_system_test.py

```diff
@@ -27,11 +27,13 @@
 import storage
 import system_test_utils
 import translate
+import vision
 
 
 TEST_MODULES = {
     'datastore': datastore,
     'speech': speech,
+    'vision': vision,
```

Comment:
So I read `MODULES = (  # ordered from most to least stable` [here](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/7cd15697f802252ab5c4ac062a0f3c4b2b80856b/system_tests/attempt_system_tests.py#L59) and I figured that speech and vision should be similarly stable. These seemed to be in the same order so that's the only reason.